src/components/Layout.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { HashRouter as Router, Switch } from "react-router-dom";
```

Comment:
prefer using `BrowserRouter`

Reply:
Isn't it best to use HashRouter because of the 404 errors that happen if we don't have a server for the redirection?

Reply:
this issue has been resolved :)

https://github.com/vitejs/vite/pull/3239